snapcraft/plugins/catkin.py

```diff
@@ -245,12 +246,18 @@ def build(self):
 
         self.run(
             ['rm', '-f',
-             'opt/ros/' + self.options.rosversion + '/.catkin',
-             'opt/ros/' + self.options.rosversion + '/.rosinstall',
-             'opt/ros/' + self.options.rosversion + '/setup.sh',
-             'opt/ros/' + self.options.rosversion + '/_setup_util.py'],
+             'opt/ros/' + self.options.rosversion + '/.rosinstall'],
             cwd=self.installdir)
 
+        # Fix the shebang in _setup_util.py to be portable
```

Comment:
It would be nice to have a unittest for the shebang.
Aren't we doing this already in the python plugin? Maybe we can share some code, and unit test that.